.github/workflows/twister.yaml

```diff
@@ -127,17 +124,19 @@ jobs:
 
   twister-test-results:
     name: "Publish Unit Tests Results"
-    needs: twister-build
-    if: always()
-    # These permissions must be set for the EnricoMi/publish-unit-test-result-action
-    # See https://github.com/EnricoMi/publish-unit-test-result-action#permissions
+    # WARNING: This must be run on Github Hosted Action Runners, as they allow the usage of Docker
+    # An alternative would be to create a custom Docker image with Docker-in-Docker enabled
+    
+    # This version is a must, twister otherwise fails (some Python library
+    # depends on a specific version of libffi that is not present in the
+    # toolchain provided by nordic's toolchain manager).
+    runs-on: ubuntu-22.04
```

Comment:
This is now not true. The original comment was referring to the ubuntu-20.04, and that was for the `twister-build` job.

Please place this comment in the docker image inside irnas-runners-software repo, as our self-hosted will be running ubuntu-20.04, so that is the most appropriate place for it (since ubuntu-22.04 and nrftutil-toolchain-manager seem to be incompatible for now).

Reply:
Also, above comment starting with `WARNING` is also not true since job `twister-test-results` uses `ubuntu-22.04` instead of `self-hosted`. Am I missing something here?

Reply:
I see now. Also, I have reworded the warning to be more concise.